File: 3-Chapter-4.1/Revision_Material/four.cs
using System;
namespace CSharpWork
{
    public class Four
    {
        public Four()
        {
            int one = IntPrompt();
            int two = IntPrompt();
            int three = IntPrompt();

            int value = Largest(one, two, three);
            Output(value);
        }

        private static int Largest(int one, int two, int three)
        {
            if (one > two && one > three)
            {
                return one;
            }
            else if (two > one && two > three)
            {
                return two;
            }
            else if (three > one && three > two)
            {
                return three;
            }
            else
            {
                return 0;
            }
        }

        static void Output(int value)
        {
            Console.WriteLine(value);
        }

        static int IntPrompt()
        {
            Console.WriteLine("Enter an integer:");
            return int.Parse(Console.ReadLine());
        }
    }
}

File: 3-Chapter-4.1/Revision_Material/four.py
one = int(input("Enter an integer:"))
two = int(input("Enter an integer:"))
three = int(input("Enter an integer:"))


def largest(first, second, third):
    if first > second and first > third:
        return first
    elif second > first and second > third:
        return second
    elif third > first and third > second:
        return third
    else:
        return 0


def output(value):
    print(value)


value = largest(one, two, three)

output(value)
